docs/schemaform/form-config.md

```diff
@@ -196,6 +196,13 @@ In addition to the uiSchema options listed in the library [https://github.com/mo
     labels: {
       chapter30: 'A readable description (Chapter 30)'
     },
+
+    // This is a map of values to a component, some text, or some jsx. If your field
+    // is a radio widget, the content here will be shown underneath the radio button
+    // for that value when it is selected.
+    nestedContent: {
```

Comment:
This is only for radio buttons. They're a single field/widget, so the only way to put content in between the options is some kind of option the widget can read.